### ISSUE TITLE
Error in BE

### ISSUE THREAD

**Issue by HenryVolkmer** [richardhj/contao-isotope_attribute_radio_image#1]
UPDATE:
dont appears in 3af503b9

---

Hi,

when create new Attribute and select type "radioimage", this error occures:

```
Fatal error: Uncaught exception Error with message Using $this when not in object context thrown in composer/vendor/isotope/isotope-core/system/modules/isotope/library/Isotope/Model/Attribute/SelectMenu.php on line 42
#0 composer/vendor/richardhj/contao-isotope_attribute_radio_image/contao/src/Isotope/Model/Attribute/RadioImage.php(75): Isotope\Model\Attribute\SelectMenu::saveToDCA(Array)
#1 composer/vendor/isotope/isotope-core/system/modules/isotope/library/Isotope/Backend/Product/DcaManager.php(118): Isotope\Model\Attribute\RadioImage->saveToDCA(Array)
#2 composer/vendor/isotope/isotope-core/system/modules/isotope/library/Isotope/Backend/Product/DcaManager.php(40): Isotope\Backend\Product\DcaManager->addAttributes()
#3 system/modules/core/library/Contao/DcaLoader.php(97): Isotope\Backend\Product\DcaManager->initialize('tl_iso_product')
#4 system/modules/core/library/Contao/Controller.php(1298): Contao\DcaLoader->load(true)
#5 composer/vendor/isotope/isotope-core/system/modules/isotope/library/Isotope/Backend/Attribute/Callback.php(103): Contao\Controller::loadDataContainer('tl_iso_product', true)
#6 system/modules/core/drivers/DC_Table.php(2042): Isotope\Backend\Attribute\Callback->updateDatabase(Object(Contao\DC_Table))
#7 composer/vendor/isotope/isotope-core/system/modules/isotope/library/Isotope/BackendModule/BackendOverview.php(222): Contao\DC_Table->edit()
#8 composer/vendor/isotope/isotope-core/system/modules/isotope/library/Isotope/BackendModule/BackendOverview.php(68): Isotope\BackendModule\BackendOverview->getModule('attributes')
#9 system/modules/core/classes/Backend.php(423): Isotope\BackendModule\BackendOverview->generate()
#10 system/modules/core/controllers/BackendMain.php(131): Contao\Backend->getBackendModule('iso_setup')
#11 contao/main.php(20): Contao\BackendMain->run()
#12 {main}
```

**Comment by richardhj** [comment]
Released v1.0.1.
Regards!

**Comment by GMTeams** [comment]
**Contao 3.5.30
contao-isotope_attribute_radio_image v1.0.1
Isotope 2.4.4**

`Pflichtfeld Optionen*

Fatal error: Uncaught exception Error with message Using $this when not in object context thrown in composer/vendor/isotope/isotope-core/system/modules/isotope/library/Isotope/Model/Attribute/SelectMenu.php on line 29
#0 system/modules/isotope_attribute_radio_image/src/Isotope/Model/Attribute/RadioImage.php(59): Isotope\Model\Attribute\SelectMenu::prepareOptionsWizard(Object(MultiColumnWizard), Array)
#1 composer/vendor/isotope/isotope-core/system/modules/isotope/library/Isotope/Backend/Attribute/OptionsWizard.php(65): Isotope\Model\Attribute\RadioImage->prepareOptionsWizard(Object(MultiColumnWizard), Array)
#2 composer/vendor/menatwork/contao-multicolumnwizard/system/modules/multicolumnwizard/MultiColumnWizard.php(337): Isotope\Backend\Attribute\OptionsWizard->getColumns(Object(MultiColumnWizard))
#3 system/modules/core/library/Contao/Widget.php(684): MultiColumnWizard->generate()
#4 system/modules/core/templates/backend/be_widget.html5(3): Contao\Widget->generateWithError(true)
#5 system/modules/core/library/Contao/BaseTemplate.php(88): include('/kunden/12345_...')
#6 system/modules/core/library/Contao/Widget.php(630): Contao\BaseTemplate->parse()
#7 system/modules/core/classes/DataContainer.php(557): Contao\Widget->parse()
#8 system/modules/core/drivers/DC_Table.php(1943): Contao\DataContainer->row('{attribute_lege...')
#9 composer/vendor/isotope/isotope-core/system/modules/isotope/library/Isotope/BackendModule/BackendOverview.php(222): Contao\DC_Table->edit()
#10 composer/vendor/isotope/isotope-core/system/modules/isotope/library/Isotope/BackendModule/BackendOverview.php(68): Isotope\BackendModule\BackendOverview->getModule('attributes')
#11 system/modules/core/classes/Backend.php(423): Isotope\BackendModule\BackendOverview->generate()
#12 system/modules/core/controllers/BackendMain.php(131): Contao\Backend->getBackendModule('iso_setup')
#13 contao/main.php(20): Contao\BackendMain->run()
#14 {main}`

Having the same error - but only on "Optionen-Assistent". In "Optionen-Manager" it works, but with "Optionen-Manager" I don't have the possibility to auto generate Variants (nothing found). When I try to manually add a variant, there is an error occuring:

`Warning: Illegal string offset 'value' in system/modules/core/widgets/SelectMenu.php on line 142
#0 system/modules/core/widgets/SelectMenu.php(142): __error(2, 'Illegal string ...', '/kunden/12345_...', 142, Array)
#1 system/modules/core/library/Contao/Widget.php(684): Contao\SelectMenu->generate()
#2 system/modules/core/templates/backend/be_widget.html5(3): Contao\Widget->generateWithError(true)
#3 system/modules/core/library/Contao/BaseTemplate.php(88): include('/kunden/12345_...')
#4 system/modules/core/library/Contao/Widget.php(630): Contao\BaseTemplate->parse()
#5 system/modules/core/classes/DataContainer.php(557): Contao\Widget->parse()
#6 composer/vendor/isotope/isotope-core/system/modules/isotope/drivers/DC_ProductData.php(598): Contao\DataContainer->row('inherit,{genera...')
#7 system/modules/core/classes/Backend.php(650): DC_ProductData->edit()
#8 system/modules/core/controllers/BackendMain.php(131): Contao\Backend->getBackendModule('iso_products')
#9 contao/main.php(20): Contao\BackendMain->run()
#10 {main}`

Thanks for this nice extension - it's very useful for color variants!
